### ISSUE TITLE
ZenPacks.community.DistributedCollectors-2.0.1.egg & Zenoss 4.2.5

### ISSUE THREAD

**Issue by marciosilvarj** [epuzanov/ZenPacks.community.DistributedCollectors#8]
Hi! I am new to Zenoss. I have it installed and set up. It's working fine at my site. I am trying to setup distributed collectors but I am finding too many errors. I am following a tutorial created on Sep 14, 2009 - Last modified on Feb 28, 2012. Some files are in different folders and some configuration files have different syntax. It makes quite hard to follow for people like me that is learning how to setup Zenoss. Can I have some help?
Kind Regards,
Marcio

**Comment by mhaswell-resilient** [comment]
Please try the latest version and see if that helps.

**Comment by marciosilvarj** [comment]
I made it work. Tks.
